### ISSUE TITLE
Serve sitemap pages over https

### ISSUE THREAD

**Issue by AndrewHaine** [justkidding96/statamic-aardvark-seo#121]
> this may be a separate issue but we're also having problems with sitemap erroring in production due to it being rendered in http rather than https. Setting the APP_URL to https does not help, nor does enforcing https in the AppServiceProvider.

> Looking into this further, I've found that the addon serves via http on servers behind load balancers. Are you able to add some configuration that forces sitemap.xml to serve https? At the moment it is only the aardvark seo tools which are being served in http

_Originally posted by @AdD959 in https://github.com/WithCandour/statamic-aardvark-seo/issues/113#issuecomment-1069085999_

**Comment by krsilas** [comment]
I'm currently struggling with this as well. Is there any way to fix it?

**Comment by AdD959** [comment]
I ran into this issue again 2 years later with another site. Any chance of this being addressed?

**Comment by krsilas** [comment]
@AdD959 For me the issue was actually not really related to aardvark-seo. I'm running the laravel app behind a proxy and I had to configure a trusted proxy (https://laravel.com/docs/10.x/requests#configuring-trusted-proxies)

**Comment by AdD959** [comment]
@krsilas you're right and I believe this was the solution I also found at the time, thanks for that! Good to have a reference here if anyone else stumbles upon the issue.
